lectures/views.py

```diff
@@ -1,3 +1,95 @@
-from django.shortcuts import render
+import boto3
 
-# Create your views here.
+from uuid          import uuid4
+from django.http   import JsonResponse
+from django.views  import View
+
+from users.models    import User
+from lectures.models import Lecture, LectureImage
+from core.decorator  import signin_decorator
+from my_settings     import AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY
+
+class FileView(View):
+    s3_client = boto3.client(
+        's3',
+        aws_access_key_id     = AWS_ACCESS_KEY_ID,
+        aws_secret_access_key = AWS_SECRET_ACCESS_KEY
+    )
+    @signin_decorator
+    def post(self, request):
+        try:
+            profile  = request.FILES['profile']
+            user     = request.user
+            users    = User.objects.filter(id=user.id)
+           
+            BUCKET_ADDRESS    = 'https://woosbucket.s3.ap-northeast-2.amazonaws.com'
+            BUCKET_DIRECTORY  = 'thumnail'
+            BUCKET_DIRECTORY2 = 'image'
+            BUCKET_DIRECTORY3 = 'profile'
+            url          = f"{BUCKET_DIRECTORY}/{str(uuid4())}"
+            urls         = f"{BUCKET_ADDRESS}/{url}"
+            profile_url  = f"{BUCKET_DIRECTORY3}/{str(uuid4())}"
+            profile_urls = f"{BUCKET_ADDRESS}/{profile_url}"
```

Comment:
```suggestion
            BUCKET_ADDRESS    = 'https://woosbucket.s3.ap-northeast-2.amazonaws.com'
            BUCKET_THUMNAIL  = 'thumnail'
            BUCKET_IMAGE = 'image'
            BUCKET_PROFILE = 'profile'
            
            profile  = request.FILES['profile']
            user     = request.user
           
            url          = f"{BUCKET_DIRECTORY}/{str(uuid4())}"
            urls         = f"{BUCKET_ADDRESS}/{url}"
            profile_url  = f"{BUCKET_DIRECTORY3}/{str(uuid4())}"
            profile_urls = f"{BUCKET_ADDRESS}/{profile_url}"
```
1. BUCKET_DIRECTORY1,2 가 아닌 명확하게 변수명을 지정하는 것이 좋습니다.
2. BUCKET 관련 변수는 my_settings.py에 옮기는 것도 좋습니다.
3. user를 이미 알고 있는데, users를 가져올 이유가 없을 것 같습니다.
4. url과 urls, profile_url과 profile_urls가 각 각 무엇을 의미하는지 파악이 잘 안 됩니다. 불필요하게 나줘서 변수를 만든 것 같습니다.